### PR TITLE
chore: add a fixed sponsor block

### PR DESCRIPTION
**Pull request by boojack**
image

### CODE REVIEW

**Review by tianzhou — COMMENTED** [review]
Please update layout per offline discussion

**Comment by boojack** [comment]
PTAL

File: src/components/SponsorBanner.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      <p class="mt-2 text-xs">
+        Database DevOps and CI/CD for MySQL, PG, ClickHouse, Snowflake, TiDB
+      </p>
+    </a>
+    <a
+      href="mailto:star@bytebase.com?subject=I'm interested in star-history"
+      target="_blank"
+      class="w-full mt-2 p-2 text-center bg-gray-50 text-xs leading-6 text-gray-400 rounded hover:underline hover:text-blue-600"
+      >Your logo<br class="xl:hidden" />(Become Sponsor)</a
+    >
```
Comment (tianzhou) [inline src/components/SponsorBanner.vue:32]:
```suggestion
    <a
      href="mailto:star@bytebase.com?subject=I'm interested in star-history"
      target="_blank"
      class="w-full mt-2 p-2 text-center bg-gray-50 text-xs leading-6 text-gray-400 rounded hover:underline hover:text-blue-600"
      >Your logo</a
    >
```

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**